[14.0][DOC]Adjust the page migration_details: (#2621)
OCA/OpenUpgrade: doc/source/migration_details.rst

(or openerp-server) directly. You will want to do this when you are working on
developing migration scripts for uncovered modules.

1. Get the code from OpenUpgrade and dependencies
*************************************************

OpenUpgrade
...........

Make the `openupgrade_framework` and the `openupgrade_scripts` modules
available in the addons path in the Odoo instance of the new version.

Or, for older versions: check out the OpenUpgrade source code from Github
for the branches you need. Each branch migrates to its version from the
previous version, so branch 13.0 migrates from 12.0 to 13.0. If you are
migrating across multiple versions, you need to run each version of
OpenUpgrade in order. Skipping versions is not supported.

The OpenUpgrade repository includes both `openupgrade_framework` and
`openupgrade_scripts`:

https://github.com/OCA/openupgrade

openupgradelib
..............

* When installing the openupgradelib make sure you check out the latest version
  from github to get the latest updates and fixes::

>     pip install git+git://github.com/OCA/openupgradelib.git

Reviewer: ```suggestion
    pip install git+https://github.com/OCA/openupgradelib.git@master#egg=openupgradelib
```

2. Check coverage of the migration scripts for your installed modules
*********************************************************************

Check if there are migration scripts provided for the set of modules that
are installed in your Odoo database. If there are modules for which no
migration scripts have been developed yet, your migration may fail or the
integrity of your database may be lacking. Check the module coverage in
this documentation under :doc:`status` and refer to the :doc:`development`
documentation to add the missing migration scripts.

3. Make a copy of the database to migrate
*****************************************

Decide which database you are going to upgrade. You absolutely *must* make a
backup of your live database before you start this process!

4. Adjust the configuration for Odoo and OpenUpgrade
****************************************************

Edit the configuration files and command line parameters to point to the
database you are going to upgrade. The recommended command line parameters are

> the ```--update all --stop-after-init --load=web,openupgrade_framework`` flags.

Reviewer: ```suggestion
the ```--update all --stop-after-init --load=base,web,openupgrade_framework`` flags.
```

Author: Is that really necessary?
If it is, then we probably should also update this README as well:

https://github.com/OCA/OpenUpgrade/blob/14.0/openupgrade_framework/README.rst

Reviewer: Doing that here https://github.com/OCA/OpenUpgrade/pull/2629

(or openerp-server) directly. You will want to do this when you are working on
developing migration scripts for uncovered modules.

1. Get the code from OpenUpgrade and dependencies
*************************************************

OpenUpgrade
...........

Make the `openupgrade_framework` and the `openupgrade_scripts` modules
available in the addons path in the Odoo instance of the new version.

Or, for older versions: check out the OpenUpgrade source code from Github
for the branches you need. Each branch migrates to its version from the
previous version, so branch 13.0 migrates from 12.0 to 13.0. If you are
migrating across multiple versions, you need to run each version of
OpenUpgrade in order. Skipping versions is not supported.

> OpenUpgrade repository includes both `openupgrade_framework`and `openupgrade_scripts`:

Reviewer 2: Backtick confusion

![image](https://user-images.githubusercontent.com/1033124/111758177-768a8f80-889c-11eb-9060-a12011a28113.png)

```suggestion
The OpenUpgrade repository includes both `openupgrade_framework` and `openupgrade_scripts`:
```

https://github.com/OCA/openupgrade

openupgradelib
..............

* When installing the openupgradelib make sure you check out the latest version
  from github to get the latest updates and fixes::

    pip install git+git://github.com/OCA/openupgradelib.git

2. Check coverage of the migration scripts for your installed modules
*********************************************************************

Check if there are migration scripts provided for the set of modules that
are installed in your Odoo database. If there are modules for which no
migration scripts have been developed yet, your migration may fail or the

> integrity of your database may me lacking. Check the module coverage in

Reviewer 2: ```suggestion
integrity of your database may be lacking. Check the module coverage in
```

> this documentation in :doc:`status` and refer to the :doc:`development`

Reviewer 2: ```suggestion
this documentation under :doc:`status` and refer to the :doc:`development`
```

> documentation to add the missing migration modules.

Reviewer 2: ```suggestion
documentation to add the missing migration scripts.
```

3. Make a copy of the database to migrate
*****************************************

Decide which database you are going to upgrade. You absolutely *must* make a
backup of your live database before you start this process!

It is easy to do through the database manager of your old odoo

> `{your.old.odoo}/web/database/manager`

Reviewer 2: Maybe it is me but I associate using the database manager for backups with a lower knowledge level than needed to run OpenUpgrade properly. I would personally leave this out because if people need us to tell them how to make a proper backup of their setup, their voyage in OpenUpgrade will not end well.

Author: I imagine the same :)

4. Adjust the configuration for Odoo and OpenUpgrade
****************************************************

Edit the configuration files and command line parameters to point to the

> database you are going to upgrade. Run Odoo with the ``--update all

Reviewer 2: ```suggestion
database you are going to upgrade. The recommended command line parameters are the ```--update all
```
